src/pages/RepoPage/FailedTestsTab/FailedTestsPage/hooks/useInfiniteTestResults/useInfiniteTestResults.tsx

```diff
@@ -127,6 +163,11 @@ interface UseTestResultsArgs {
   repo: string
   filters?: {
     branch?: string
+    flags?: string[]
+    interval?: MeasurementInterval
+    parameter?: TestResultsFilterParameterType
+    term?: string
+    test_suites?: string[]
```

Comment:
I was wondering why this one was snake case - guess it's that way in `api` - https://github.com/codecov/codecov-api/blob/7857053efdb545165cbce4ffc0bd9526d82a73aa/graphql_api/types/inputs/test_results_filters.graphql#L4

Seems ok as is - if we ever have time to clean it up can grab it later

Reply:
Here's the issue [The test_suites key in the test results filter needs to be camel-cased to align with naming standards #2878](https://github.com/codecov/engineering-team/issues/2878)